src/x11/x.c

```diff
@@ -943,7 +943,11 @@ static void x_shortcut_init(struct keyboard_shortcut *ks)
 }
 
 int x_get_scale(void) {
-        return 1;
+        const struct screen_info *scr_info = get_active_screen();
+        int scale = round(scr_info->dpi/96.);
```

Comment:
Throwing away fractional scaling this easily seems a waste. dunst already supports fractional font scaling. Can't we do the same for geometry?

Reply:
The problem is that wayland only supports integer scaling. It would be more work to change that to fractional scaling. For now I've just repurposed the scaling code from wayland.

> dunst already supports fractional font scaling. Can't we do the same for geometry?

In what way do we support fractional font scaling? By setting the font size?